bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -2694,7 +2694,7 @@ lnd/clm2/surfdata_esmf/NEON/surfdata_1x1_NEON_TOOL_hist_78pfts_CMIP6_simyr2000_c
 <!-- Defaults for excess ice                    -->
 <!-- =========================================  -->
 
-<use_excess_ice>.false.</use_excess_ice>
+<use_excess_ice>.true.</use_excess_ice>
```

Comment:
@slevis-lmwg we just want this on for clm6_0 physics and off otherwise.

Reply:
Yes, thank you for catching that @ekluzek 

Also a question:
I'm having trouble identifying any finidat lines that I should change, i.e. corresponding to 1850 and clm6. So instead I'm adding a new entry for f09-1850-clm6.

Reply:
I'll help with that one. This is an example of why we need to do #2169. The XML has gotten really duplicative and complex with the addition of clm6_0 and cam7.0.

Reply:
thank you!